pkg/mcproducer/mcproducer.go

```diff
@@ -71,6 +75,11 @@ func ProduceMachineConfig(machineConfigName,
 		return "", err
 	}
 
+	templateParams["WaitForNetworkDispatcherContents"], err = executeIntoBase64(scriptWaitForNetworkDispatcher, templateParams)
```

Comment:
Does this need to be rendered as a template? I don't see any variable in it, we could probably just encode it to base64.

Reply:
Yes, but we already have a function for encoding, so even if there are no template variables in it, i prefer to use existing function, then defining a new one, or calling an base64 API directly